packages/edit-site/src/components/add-new-pattern/index.js

```diff
@@ -56,8 +56,14 @@ export default function AddNewPattern() {
 			addNewTemplatePartLabel: getPostType( TEMPLATE_PART_POST_TYPE )
 				?.labels?.add_new_item,
 			// Blocks refers to the wp_block post type, this checks the ability to create a post of that type.
-			canCreatePattern: canUser( 'create', 'blocks' ),
-			canCreateTemplatePart: canUser( 'create', 'template-parts' ),
+			canCreatePattern: canUser( 'create', {
+				kind: 'postType',
+				name: 'wp_block',
+			} ),
+			canCreateTemplatePart: canUser( 'create', {
+				kind: 'postType',
+				name: 'wp_template_part',
```

Comment:
Could we use `TEMPLATE_PART_POST_TYPE` here, and in similar in other places?

Reply:
Update to use constants where it made sense - where the constant was available.